test/unit/region.spec.js

```diff
@@ -3,6 +3,7 @@ import Backbone from 'backbone';
 import Events from '../../src/mixins/events';
 import Region from '../../src/region';
 import View from '../../src/view';
+import CollectionView from '../../src/collection-view';
```

Comment:
We should also test for collectionview functionality in the relevant collectionview spec file which is probably with `addChildView` and `detachChildView`